src/grt/src/fastroute/src/utility.cpp

```diff
@@ -2488,6 +2490,13 @@ void FastRouteCore::saveCongestion(const int iter)
   std::vector<CongestionInformation> congestionGridsV, congestionGridsH;
   if (!h_edges_.empty() && !v_edges_.empty()) {
     getCongestionGrid(congestionGridsV, congestionGridsH);
+
+    std::mt19937 g;
+    const int seed = 42;
+    g.seed(seed);
+
+    utl::shuffle(congestionGridsV.begin(), congestionGridsV.end(), g);
+    utl::shuffle(congestionGridsH.begin(), congestionGridsH.end(), g);
```

Comment:
There is still an ordering for V markers then H markers.  With many markers the V will hit the limit and block the H.

Reply:
Is that right? When I open the GUI, I see two different tabs for V and H, and both of them have 10000 occurrences.

Reply:
I didn't realize they are separate categories